impeller/aiks/canvas_recorder.h

```diff
@@ -16,6 +16,7 @@ namespace impeller {
 /// golden tests can be written at a higher level, migrate these to
 /// flutter::DisplayListOpType.
 enum CanvasRecorderOp : uint16_t {
+  // NOLINTBEGIN(readability-identifier-naming)
```

Comment:
You don't just want to rename them?

Reply:
I could. It caused problems with a macro used to map to something else, but maybe I was just tired at the time. I can give it a second shot.

Reply:
Ok I was just being lazy, this seems fine :)